internal/runners/invite/invite.go

```diff
@@ -63,7 +64,9 @@ func (i *invite) Run(params *Params) error {
 		}
 	}
 
-	emails := strings.Split(params.EmailList, ",")
+	re := regexp.MustCompile(",,+")
+	emailList := strings.Trim(re.ReplaceAllString(params.EmailList, ","), ",")
+	emails := strings.Split(emailList, ",")
```

Comment:
Would iterating through `emails` and removing any blank entries cover more potential combinations than the regex?

Reply:
I don't think so. We're splitting on ',' so the only way to get a blank e-mail is if there is a leading, trailing, or double ','. All of those cases have been covered.